src/ca11y/index.js

```diff
@@ -370,8 +388,11 @@ Ca11y.pickers = []
  * Ca11y.init is used to instantiate multiple datepickers on a page simultaneously.
  **/
 Ca11y.init = function(selector, options) {
-  const datePickers = Array.from(document.querySelectorAll(selector) || 0)
-  datePickers.forEach(input => new Ca11y(input, options))
+  const datePickers = document.querySelectorAll(selector)
+  for (let i = 0; i < datePickers.length; i++) {
+    const input = datePickers[i]
+    new Ca11y(input, options)
+  };
 }
```

Comment:
would it be worth returning a list of these `Ca11y` instances? Maybe not. How useful do you think it would be?